File: Library/core/src/main/java/com/emedinaa/core/media/ImageLoader.java
package com.emedinaa.core.media;

import android.widget.ImageView;

/**
 * @author : Eduardo Medina
 * @see : https://developer.android.com/index.html
 * @since : 8/11/18
 */
public interface ImageLoader {

    void load(String url, ImageView imageView);
    void loadCircle(String url, ImageView imageView);
    void loadLocal(String path, ImageView imageView);
}

File: RestaurantApp/app/src/main/java/com/emedinaa/kotlinapp/util/ImageLoader.kt
package com.emedinaa.kotlinapp.util

import android.widget.ImageView

interface ImageLoader {

    fun load(url: String, imageView: ImageView)
    fun loadWithResize(url: String, imageView: ImageView,resize:Int)
    fun loadCircle(url: String, imageView: ImageView)
    fun loadLocal(path: String, imageView: ImageView)
}